src/QsCompiler/CompilationManager/CompilationUnit.cs

```diff
@@ -452,8 +454,9 @@ private QsCallable GetImportedCallable(CallableDeclarationHeader header)
                 var specSignature = specHeader.Kind.IsQsControlled || specHeader.Kind.IsQsControlledAdjoint 
                     ? SyntaxGenerator.BuildControlled(header.Signature) 
                     : header.Signature;
-                return new QsSpecialization(specHeader.Kind, header.QualifiedName, specHeader.Attributes, specHeader.SourceFile, specLocation, 
-                    specHeader.TypeArguments, specSignature, implementation, specHeader.Documentation, QsComments.Empty);
+                return new QsSpecialization(specHeader.Kind, header.QualifiedName, specHeader.Attributes, CapabilityLevel.Unset,
```

Comment:
One more. Sorry I left the branch in an inconsistent state!
```suggestion
                return new QsSpecialization(specHeader.Kind, header.QualifiedName, specHeader.Attributes, 
```